src/components/organisms/FormCadastro/index.tsx

```diff
@@ -79,7 +79,11 @@ export function FormCadastro(props: {
       .required(ERRORS.REQUIRED_NAME)
       .trim()
       .min(MIN_LENGTH_NAME_VALUE, ERRORS.MIN_LENGHT_NAME)
-      .max(MAX_LENGTH_NAME_VALUE, ERRORS.MAX_LENGHT_NAME),
+      .max(MAX_LENGTH_NAME_VALUE, ERRORS.MAX_LENGHT_NAME)
+      .matches(
+        /\b[A-Za-zÀ-ú][A-Za-zÀ-ú]+,?\s[A-Za-zÀ-ú][A-Za-zÀ-ú]{2,19}\b/gi,
```

Comment:
On the expressions:
> [A-Za-zÀ-ú]

Could it be changed to something like:
> [a-Zà-Ú]

Reply:
<img width="955" alt="image" src="https://user-images.githubusercontent.com/94059872/170285778-cd575589-25f4-46c2-ac95-8cfbdecd3efe.png">
I got an error when i tried testing it